DecipherNow/gm-fabric-dashboard
# fix: if a status category doesnt receive microservices its section wi…

…ll no longer be visible in the list view

Resolves #723 

## Proposed Changes

  -Adding a check to see if microservices are present under a heading key, if not, that section should not be rendered in the list view
  
## How to Test

  - /FabricMainView/scene/ServicesListView/ServicesListView.js
  - around line 48, add something similar to dataGroupedByHeader['down']='';
  - Navigate to list view after, no empty section should be visible.
